interp/src/interpreter/interpret_control.rs

```diff
@@ -1,34 +1,62 @@
 //! Inteprets a control in a component.
 
-use super::interpret_group::interpret_group;
+use super::interpret_group::{finish_group_interpretation, interpret_group};
 use crate::environment::Environment;
 use calyx::{errors::FutilResult, ir};
 
-/// Helper function to evaluate control
 pub fn interpret_control(
     ctrl: &ir::Control,
-    comp: &ir::Id,
+    continuous_assignments: &[ir::Assignment],
+    env: Environment,
+) -> FutilResult<Environment> {
+    match ctrl {
+        ir::Control::Seq(ir::Seq { stmts, .. }) => {
+            if stmts.len() == 1 {
```

Comment:
Can you elaborate on what gets made impossible, I'm not sure I understand

Reply:
It ensures that there are no `seq { ... }` blocks with exactly one control statement in them. If this is the case, it removes the `seq` block altogether because every control statement allows for nesting of one statement within it.